mix.exs

```diff
@@ -44,7 +48,8 @@ defmodule Payeezy.Mixfile do
       {:bypass, "~> 0.6.0", only: [:test, :dev]},
       {:excoveralls, "~> 0.7", only: :test},
       {:plug, "~> 1.3", only: [:test, :dev]},
-      {:ex_doc, ">= 0.0.0", only: :dev}
+      {:ex_doc, ">= 0.0.0", only: :dev},
+      {:dialyxir, "~> 1.0", only: [:dev], runtime: false}
```

Comment:
You should update the `application` function in this module to:

```
  def application do
    [extra_applications: [:logger]]
  end
```

With newer versions of elixir it is recommended that you don't specify which applications to run because elixir is good at figuring it out automatically. Instead you specify `extra_applications` for specifying anything elixir can't figure out by itself which is for the most part apps that are part of erlang and not elixir such as `:logger`. When I made this change locally it got rid of a bunch of compilation warnings.